pkg/kubelet/cm/cpumanager/state/state_checkpoint.go

```diff
@@ -88,6 +87,7 @@ func (sc *stateCheckpoint) restoreState() error {
 		return fmt.Errorf("could not parse default cpu set %q: %v", checkpoint.DefaultCPUSet, err)
 	}
 
+	tmpContainerCPUSet := cpuset.NewCPUSet()
```

Comment:
```suggestion
	var tmpContainerCPUSet cpuset.CPUSet
```

Reply:
For both this file, and the one below, I believe we can avoid the method call to `NewCPUSet` by just declaring the variable.

Reply:
Adapted as suggested, now it looks way better, thanks :)